package.json

```diff
@@ -382,6 +382,7 @@
     "regenerator-runtime": "0.13.9",
     "reselect": "4.1.6",
     "rst2html": "github:thoward/rst2html#990cb89f2a300cdd9151790be377c4c0840df809",
+    "rudder-sdk-js": "^2.13.0",
```

Comment:
if we're just using this for types, can this be a `devDep`? 🤔

Reply:
Yeah good question. I didn't exactly know the best place to put it, but i think dev deps might be better? Either way, it's not like we actually run/use a 'production' node_modules anyway.

Reply:
yep true, it doesn't really matter either way 🙂